src/kits/daisy/controller_node.cpp

```diff
@@ -0,0 +1,100 @@
+#include <ros/ros.h>
+#include <ros/console.h>
+#include <ros/package.h>
+
+#include <geometry_msgs/Twist.h>
+#include <std_msgs/Bool.h>
+
+#include "input/mobile_io.hpp"
+
+// Helper to apply a "dead zone" to an input value
+float applyDeadZone(float raw, float dead_zone) {
+  if (std::abs(raw) < dead_zone)
+    return 0;
+  // Scale from [dead_zone, 1] to [0, 1] (and same for [-1, -dead_zone]
+  if (raw > 0)
+    return (raw - dead_zone) / (1 - dead_zone);
+  return (raw + dead_zone) / (1 - dead_zone);
+}
+
+int main(int argc, char ** argv) {
+
+  // Initialize ROS node
+  ros::init(argc, argv, "controller");
+  ros::NodeHandle node;
+
+  // Create mobile IO connection
+  auto io = hebi::MobileIO::create("HEBI", "Mobile IO");
```

Comment:
we can fix this post-merge -- but this should be updated to `Daisy` / `mobileIO` at some point